content/ko/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -86,12 +86,36 @@ curl http://localhost:8080/api/
 
 ### kubectl proxy를 사용하지 않음
 
-기본 서비스 어카운트의 토큰을 얻어내려면 `kubectl describe secret...`을 grep/cut과 함께 사용한다.
+`kubectl apply` 및 `kubectl describe secret...` 명령과 grep/cut을 활용하여 기본 서비스 어카운트의 토큰을 생성한다.
+
+먼저, 기본 서비스어카운트를 위한 토큰을 요청하는 시크릿을 생성한다.
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: default-token
+  annotations:
+    kubernetes.io/service-account.name: default
+type: kubernetes.io/service-account-token
+EOF
+```
+
+다음으로, 토큰 컨트롤러가 해당 시크릿에 토큰을 채우기를 기다린다.
+
+```shell
+while ! kubectl describe secret default-token | grep -E '^token' >/dev/null; do
+  echo "waiting for token..." >&2
```

Comment:
(small, totally optional, nit)
```suggestion
  echo "waiting for token..." >&2 # 토큰을 기다리다
```

Reply:
Hi @sftim 
Thank you for the suggestion ! :)

In the case of Korean localization, we agreed to avoid localization of the content in `code`. (we only localize English comments in `code`)